### PR TITLE
Support for networks connected

### PR DESCRIPTION
**Pull request by KarlStraussberger**
If a network is connected it creates an entry for each client and also for routes which are pushed. These entries need to be filtered. Doing this with netmask

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "maxmind": "4.1.1",
     "moment": "2.24.0",
     "nconf": "0.10.0",
+    "netmask": "^1.0.6",
```
Comment (AuspeXeu) [inline package.json:44]: Can you make this a fixed version please?

File: openvpn.js [hunk 2]
```diff
@@ -29,12 +30,16 @@ const mkClient = (data, original = {}) => {
...
-  constructor(host, port, pwd = null) {
+  constructor(host, port, pwd = null, netmask = null) {
     super()
     this.state = STATE.idle
     this.host = host
     this.port = port
     this.pwd = pwd
+    this.netmask = new Netmask ('0.0.0.0/0')
+    if (netmask != null) {
+            this.netmask = new Netmask(netmask);
```
Comment (AuspeXeu) [inline openvpn.js:41]: Some sort of validation would be great here. Maybe a regex?

Comment (KarlStraussberger) [reply openvpn.js]: Don't want to reimplement Netmask checks. Catch exception instead.

**Comment by AuspeXeu** [comment]
Can you make a small example scenario when this filter is needed? Thanks!

**Comment by KarlStraussberger** [comment]
A branch office connects via tunnel to my site. The client config is
`ccd/client`
```
ifconfig-push 192.168.10.65 255.255.255.0
iroute 192.168.178.0 255.255.255.0
iroute 192.168.90.215 255.255.255.255
iroute 192.168.90.211 255.255.255.255
push "route 192.168.2.0 255.255.255.0"
push "route 192.168.90.0 255.255.255.0"
```

This produces `openvpn-status.log`
```
OpenVPN CLIENT LIST
Updated,Mon Apr  6 19:26:39 2020
Common Name,Real Address,Bytes Received,Bytes Sent,Connected Since
client,91.1.1.1:38458,535519,447378,Mon Apr  6 16:58:03 2020
client,91.1.1.2:38459,4711,815Mon Apr  6 18:58:15 2020
ROUTING TABLE
Virtual Address,Common Name,Real Address,Last Ref
192.168.178.2C,client,91.1.1.1:38458,Mon Apr  6 19:26:37 2020
192.168.10.65,client,91.1.1.1:38458,Mon Apr  6 19:26:31 2020
192.168.10.12,client2,91.1.1.2:38459,Mon Apr  6 19:26:31 2020
192.168.90.211,client,91.1.1.1:38458,Mon Apr  6 19:26:33 2020
192.168.178.0/24,client,91.1.1.1:38458,Mon Apr  6 16:58:04 2020
192.168.178.1C,client,91.1.1.1:38458,Mon Apr  6 19:26:24 2020
192.168.90.215,client,91.1.1.1:38458,Mon Apr  6 19:26:37 2020
GLOBAL STATS
Max bcast/mcast queue length,0
END
```

The entries with virtual address ending "C" are automatically created by devices connecting through the tunnel. The iroute entries in client config are copied directly to the routing table. All direct clients can be identified by netmask 192.168.10.0/24. To filter them from the routing table we need the additional argument for the server. The netmask is also part of the `server.conf`:
```
...
mode server
server **192.168.10.0 255.255.255.0**
topology subnet
...
```

**Pull request merged**